### PR TITLE
aiobotocore 2.4.0

### PR DESCRIPTION
**Pull request by SC426**
s3fs 2020.10 --> aiobotocore 2.4.0 

Jira ticket: https://anaconda.atlassian.net/browse/PKG-154
Upstream repo: https://github.com/aio-libs/aiobotocore
Changelog: https://github.com/aio-libs/aiobotocore/blob/2.4.0/CHANGES.rst

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,11 +8,10 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 5fd4d7aefa0896fe4dd32815ead8a52ed5ccb8016c7c5743fe8ff71c3c074120
+  sha256: f9fe0698cc497861bdb54cd16161c804031f758ada9480c35540f20c0c078385
 
 build:
   number: 0
-  noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:15]: Skip py<37, see https://github.com/aio-libs/aiobotocore/blob/fbfeb1242ab1f378f6d4fe84b5d3dacdd8874189/setup.py#L65

File: recipe/meta.yaml [hunk 2]
```diff
@@ -24,8 +23,8 @@ requirements:
   run:
     - aiohttp >=3.3.1
     - aioitertools >=0.5.1
-    - botocore >=1.23.24,<1.23.25
-    - python >=3.6
+    - botocore >=1.27.59,<1.27.60
+    - python
     - wrapt >=1.10.10
 
 test:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:31]: Please remove `python <3.10` from test/requires.

File: recipe/meta.yaml (hunk 1 above)
Comment (SC426) [reply recipe/meta.yaml:15]: This has been added

File: recipe/meta.yaml (hunk 2 above)
Comment (SC426) [reply recipe/meta.yaml:31]: This has been removed

**Review by skupr-anaconda — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
Do not forget to remove before merging: 
```
channels:
  scrass_org: s3fs
```

**Review by ELundby45 — APPROVED** [review]
LGTM

**Pull request merged**